lib/telemetry_metrics_statsd/event_handler.ex

```diff
@@ -123,4 +129,14 @@ defmodule TelemetryMetricsStatsd.EventHandler do
       end
     end)
   end
+
+  @spec sample(Metrics.t()) :: Metrics.measurement() | nil
+  defp sample(metric) do
+    rate = Keyword.get(metric.reporter_options, :sample_rate, 1.0)
```

Comment:
```suggestion
    rate = Keyword.get(metric.reporter_options, :sampling_rate, 1.0)
```

Can we use that name, or is `sample_rate` the correct wording?

Reply:
Renamed. I wondered about that, but went with Bryan's comment from https://github.com/beam-telemetry/telemetry_metrics/issues/33 since he was the last to say anything.